src/content/docs/pt-br/recipes.mdx

```diff
@@ -15,29 +15,6 @@ Veja exemplos guiados de como construir funcionalidades no Astro.
 
-Adicione o seu aqui! Consulte nosso [guia de contribuição com receitas](https://contribute.docs.astro.build/guides/recipe-writing/) para mais informações.
-
-:::note[Nota dos tradutores]
-Apesar das descrições abaixo estarem traduzidas, o conteúdo dos links é em Inglês.
-:::
-
-- [Use um nome de arquivo dinâmico ao importar imagens](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
-- [Adicione transições de página animadas com Swup](https://navillus.dev/blog/astro-plus-swup)
-- [Use UnoCSS no Astro](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
-- [Construa um índice a partir de títulos Markdown do Astro](https://kld.dev/building-table-of-contents/)
-- [Crie um plugin Remark para remover viúvas/órfãs dos seus arquivos Markdown](https://eatmon.co/blog/remove-runts-markdown/)
-- [Adicione pesquisa ao seu site com Pagefind](https://blog.otterlord.dev/post/astro-search/)
-- [Consiga o VSCode, ESLint e Prettier funcionando com Astro](https://patheticgeek.dev/blog/astro-prettier-eslint-vscode)
-- [Melhore suas builds do Astro: modificando arquivos HTML após a build](https://straffesites.com/en/blog/optimize-astro-html-post-build)
-- [Adicione modo escuro ao Astro com Tailwind CSS](https://www.kevinzunigacuellar.com/blog/dark-mode-in-astro/)
-- [Gere sitemaps localizados para seu projeto com Astro e Storyblok](https://straffesites.com/en/blog/localized-sitemap-astro-storyblok)
-- [Integre Prettier com Astro e Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
-- [Adicione um feed RSS ao Astro usando stories do Storyblok](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
-- [Usando tRPC no Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
-- [Como fazer seu website Astro multilíngue com Crowdin: Guia de Localização no Astro](https://crowdin.com/blog/2023/06/21/astro-localization-and-i18n)
-- [Adicione imagens de postagens ao seu feed RSS do Astro](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
-- [Configurando micro-frontends com Astro](https://medium.com/@sergio.a.soria/setting-up-micro-frontends-with-astro-and-ecma-script-modules-137340d2c520)
-- [Adicione autenticação com nome e senha com Lucia](https://lucia-auth.com/guidebook/sign-in-with-username-and-password/astro)
-- [Adicione OAuth do Github com Lucia](https://lucia-auth.com/guidebook/github-oauth/astro)
-- [Integrando Sentry com Astro](https://akashrajpurohit.com/blog/seamless-error-tracking-integrating-sentry-with-astro/)
-- [Configurando Páginas de Rascunho de Forma Eficiente no Astro com Autoria de Conteúdo Orientada por Configuração](https://akashrajpurohit.com/blog/set-up-draft-pages-effectively-in-astro-with-configdriven-content-authoring/)
+import ReadMore from '~/components/ReadMore.astro'
+
+<ReadMore>Veja ainda mais receitas escritas e mantidas pela comunidade em [receitas e guias da comunidade](/pt-br/community-resources/content/#recipes-and-guides)</ReadMore>
```

Comment:
the header link (`#recipes-and-guides`) will be updated in a future PR that translates the new page.